scripts/jira-automation.py

```diff
@@ -94,6 +176,17 @@ def get_jira_label_prefix(self) -> str:
         """
         return self.__jira_label_prefix
 
+    def create_labels(self, vulnerability):
+        labels = [vulnerability.get_jira_snyk_id(), "snyk", "security"]
```

Comment:
for the labels please take a look in the doc share by Jay.

Reply:
👍

Reply:
referred the doc and made changes now